trinity/main.py

```diff
@@ -262,7 +262,8 @@ def trinity_boot(args: Namespace,
             networking_process,
             plugin_manager,
             main_endpoint,
-            event_bus
+            event_bus,
+            ev.reason
```

Comment:
Just pointing out that `lahja` provides type safety for these events which means using something like `ev.foobar` results in a `mypy` error. However, for some reason, our type safety from mypy running inside `tox` is currently degraded (see https://github.com/python/mypy/issues/5771)

Reply:
Thoughts on add a temporary new CI run to run `mypy` outside of tox in circle?

Reply:
Fixed with #1389